source/_components/plex.markdown

```diff
@@ -33,14 +33,16 @@ If your server enforces SSL connections, write "`on`" or "`true`" in the _"Use S
   <img src='{{site_root}}/images/screenshots/plex-token.png' />
 </p>
 
-You can also enable the plex platform directly by adding the following lines to your `configuration.yaml`:
+You can also enable the `plex` platform directly by adding the following lines to your `configuration.yaml`:
 
 ```yaml
 # Example configuration.yaml entry
 plex:
     token: MYSECRETTOKEN
 ```
 
+_Note_: At least one of `host` or `token` must be provided.
```

Comment:
We have a special way to add notes to our documentation, see: <https://developers.home-assistant.io/docs/en/documentation_create_page.html#html>

Reply:
Didn't see that, thanks!